libs/es.js

```diff
@@ -34,7 +34,11 @@ async function connect() {
 
   // use local client
   if (!process.env.ES_HOST) {
-    client = new elasticsearch.Client({host: 'localhost:9200'})
+    if (!process.env.ES_TEST_HOST) {
+      client = new elasticsearch.Client({host: 'localhost:9200'})
+    } else {
+      client = new elasticsearch.Client({host: process.env.ES_TEST_HOST})
+    }
   } else {
```

Comment:
Defining process.env.ES_HOST to my local development host (ES 6.0 docker image) did not work, the following error is generated when using http-aws-es:

```
Elasticsearch ERROR: 2018-07-08T20:50:40Z
  Error: Request error, retrying
  HEAD http://192.168.99.100:9200/ => Data must be a string or a buffer
      at Log.error (/sat-api/node_modules/elasticsearch/src/lib/log.js:225:56)
      at checkRespForFailure (/sat-api/node_modules/elasticsearch/src/lib/transp
ort.js:258:18)
      at done (/sat-api/node_modules/http-aws-es/connector.js:48:7)
      at <anonymous>
      at process._tickDomainCallback (internal/process/next_tick.js:228:7)
      at Function.Module.runMain (module.js:695:11)
      at startup (bootstrap_node.js:188:16)
      at bootstrap_node.js:609:3
```

If the change is OK I'll update the documentation to reference ES_TEST_HOST